### PR TITLE
Update Win2D and Maui.Graphics

### PR DESCRIPTION
**Pull request by mattleibow**
### Description of Change ###

Update Maui.Graphics and made sure all the projects get the correct OutputType.

### Additions made ###

* Adds 

### PR Checklist ###

- [ ] Targets the correct branch 
- [ ] Tests are passing (or failures are unrelated)
- [ ] Targets a single property for a single control (or intertwined few properties)
- [ ] Adds the property to the appropriate interface
- [ ] Avoids any changes not essential to the handler property
- [ ] Adds the mapping to the PropertyMapper in the handler
- [ ] Adds the mapping method to the Android, iOS, and Standard aspects of the handler
- [ ] Implements the actual property updates (usually in extension methods in the Platform section of Core)
- [ ] Tags ported renderer methods with [PortHandler]
- [ ] Adds an example of the property to the sample project (MainPage)
- [ ] Adds the property to the stub class
- [ ] Implements basic property tests in DeviceTests

#### Does this PR touch anything that might affect accessibility?
- [ ] Does this PR introduce a new control? (If yes, add an example using SemanticProperties to the SemanticsPage)
- [ ] APIs that modify focusability?
- [ ] APIs that modify any text property on a control?
- [ ] Does this PR modify view nesting or view arrangement in anyway?
- [ ] Is there the smallest possibility that your PR will change accessibility? 
- [ ] I'm not sure, please help me

If any of the above checkboxes apply to your PR, then the PR will need to provide testing to demonstrate that accessibility still works.

### CODE REVIEW

File: eng/Microsoft.Extensions.targets [hunk 1]
```diff
@@ -8,8 +8,7 @@
    <!-- Microsoft.Maui.Graphics related Packages -->
   <PropertyGroup>
     <_MicrosoftGraphicsWin2DVersion>0.8.0.21</_MicrosoftGraphicsWin2DVersion>
```
Comment (jsuarezruiz) [inline eng/Microsoft.Extensions.targets:10]: Is not the Win2D library using a -rc.1.xxx in the version?, should be?

Comment (Redth) [reply eng/Microsoft.Extensions.targets:10]: That's not available yet, BUT maybe this should use `0.8.2.23`?

File: src/Controls/samples/Controls.Sample.SingleProject/Maui.Controls.Sample.SingleProject.csproj [hunk 2]
```diff
@@ -46,7 +46,7 @@
 
   <PropertyGroup>
     <!-- Required - WinUI can't deploy in a multi-targeting environment -->
-    <AppxPackageRecipe>bin\$(Configuration)\net6.0-windows10.0.19041\win-x64\Maui.Controls.Sample.SingleProject.build.appxrecipe</AppxPackageRecipe>
+    <OutputType>WinExe</OutputType>
```
Comment (Redth) [inline src/Controls/samples/Controls.Sample.SingleProject/Maui.Controls.Sample.SingleProject.csproj:49]: This needs a condition for -windows TFM

File: eng/Microsoft.Extensions.targets (hunk 1 above)
Comment (mattleibow) [reply eng/Microsoft.Extensions.targets:10]: https://github.com/dotnet/Microsoft.Maui.Graphics/pull/162

**Review by Redth — APPROVED** [review]

**Review by jsuarezruiz — APPROVED** [review]

**Pull request merged**